webview-ui/src/panes/Charts.tsx

```diff
@@ -11,46 +12,56 @@ type ChartsProps = {
     nominal: string[];
   };
   setFilteredView: (exampleFilter: ExampleFilter) => void;
-}
+};
 
 export const Charts = (props: ChartsProps) => {
   const { dataset: rawDataset, features } = props;
 
-  const dataset = rawDataset.filter((x) => x.outcome === "passed" || x.outcome === "failed")
+  const dataset = rawDataset.filter((x) => x.outcome === "passed" || x.outcome === "failed");
 
   const coverageChart = null; // CoverageChart({ dataset });
 
-  return <div className="w-full grid grid-cols-1">
-    {coverageChart &&
-      <Card>
-        {coverageChart}
-      </Card>}
-    {[...features.nominal.map((x) =>
-      <Card key={`bucket-${x}`}>
-        <NominalChart
-          feature={x}
-          dataset={dataset}
-          viewValue={(value) => props.setFilteredView({ nominal: x, value })} />
-      </Card>
-    ),
-    ...features.ordinal.flatMap((x) =>
-      [
-        <Card key={`feature-${x}`}>
-          <OrdinalChart
-            feature={x}
-            dataset={dataset}
-            viewValue={(value) => props.setFilteredView({ ordinal: x, value })} />
-        </Card>,
-      ]),
-    ]}
-    {/* <Card className="text-sm">
+  return (
+    <div className="grid w-full grid-cols-1">
+      {coverageChart && <Card>{coverageChart}</Card>}
+      {[
+        ...features.nominal.map((x) => (
+          <Card key={`bucket-${x}`}>
+            <NominalChart
+              feature={x}
+              dataset={dataset}
+              viewValue={(value) => props.setFilteredView({ nominal: x, value })}
+            />
+          </Card>
+        )),
+        ...features.ordinal.flatMap((x) => [
+          <Card key={`feature-${x}`}>
+            <OrdinalChart
+              feature={x}
+              dataset={dataset}
+              viewValue={(value) => props.setFilteredView({ ordinal: x, value })}
+            />
+          </Card>,
+        ]),
+      ]}
+      {/* <Card className="text-sm">
       <strong>NOTE:</strong> You can add your own charts by registering "features" for your test
       inputs. Consult your PBT framework's documentation to learn more.
     </Card> */}
-    <Card>
-      <UniqueTimeChart
-        dataset={dataset}
-      />
-    </Card>
-  </div >;
-}
\ No newline at end of file
+      <Card>
+        <UniqueTimeChart dataset={dataset} />
+      </Card>
+      <Card>
+        <TimingChart
+          dataset={dataset}
+          viewValues={(examples) =>
+            props.setFilteredView({
+              subset: "Selected samples by time taken",
```

Comment:
Didn't bother putting the time range. The ExampleView shows its weakness here, because the timing breakdown will only show up in the expanded example details after a bunch of scrolling. Ideally we'd have a bespoke PrettyExample that shows different information depending on how you ended up in the example view.

Reply:
At first I just see the example
![image](https://github.com/tyche-pbt/tyche-extension/assets/3701912/ea0c7736-4be7-4ac9-893e-bf8130688d4c)

The timing info is way down at the bottom
![image](https://github.com/tyche-pbt/tyche-extension/assets/3701912/9ec05bdc-6d43-4576-8c9a-501de01739a0)

Reply:
Hmm. Yeah, I think the example view could probably be a lot more informative, but I'm not sure I have great ideas. (Also this is a weakness of the duplicate aggregation — you lose info about differences in timings between the "same" value, right?)

Reply:
Yeah that's a good point. Do you think we should fix it here or just leave it as is? I don't have a good solution for resolving that that wouldn't require overhauling PrettyExample